-1

**Failed Tests**:   Build  ClangBuild 
**Summary**: https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-e623b8/31600/summary.html
**COMMIT**: 56121db7ce38b9549929a8622926d615c1552c20
**CMSSW**: CMSSW_13_1_X_2023-03-26-0000/el8_amd64_gcc11
**User test area**: For local testing, you can use `/cvmfs/cms-ci.cern.ch/week0/cms-sw/cmssw/41191/31600/install.sh` to create a dev area with all the needed externals and cmssw changes.


## Build

I found compilation error when building: 

<pre>>> Compiling  /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_13_1_X_2023-03-26-0000/src/Geometry/EcalTestBeam/test/testEcalTBHodoscopeGeometry.cc
>> Building binary testEcalTBHodoscopeGeometry
Copying tmp/el8_amd64_gcc11/src/Geometry/EcalTestBeam/test/testEcalTBHodoscopeGeometry/testEcalTBHodoscopeGeometry to productstore area:
>> Compiling  /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_13_1_X_2023-03-26-0000/src/Geometry/EcalTestBeam/test/testEcalTBCrystalMap.cpp
/data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_13_1_X_2023-03-26-0000/src/Geometry/EcalTestBeam/test/testEcalTBCrystalMap.cpp: In function 'int main()':
/data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_13_1_X_2023-03-26-0000/src/Geometry/EcalTestBeam/test/testEcalTBCrystalMap.cpp:21:10: error: 'LogVerbatim' is not a member of 'std'; did you mean 'edm::LogVerbatim'?
   21 |     std::LogVerbatim("EcalTestBeam") << "Crystal number " << thisCrystal << " eta = " << thisEta
      |          ^~~~~~~~~~~
In file included from /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_13_1_X_2023-03-26-0000/src/Geometry/EcalTestBeam/interface/EcalTBCrystalMap.h:15,
                 from /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_13_1_X_2023-03-26-0000/src/Geometry/EcalTestBeam/test/testEcalTBCrystalMap.cpp:1:
/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02778/el8_amd64_gcc11/cms/cmssw/CMSSW_13_1_X_2023-03-26-0000/src/FWCore/MessageLogger/interface/MessageLogger.h:128:9: note: 'edm::LogVerbatim' declared here
</pre>

## Clang Build

I found compilation error while trying to compile with clang. Command used:
```
USER_CUDA_FLAGS='--expt-relaxed-constexpr' USER_CXXFLAGS='-Wno-register -fsyntax-only' scram build -k -j 32 COMPILER='llvm compile'

```


<pre>>> Local Products Rules ..... done
>> Creating project symlinks
>> Entering Package Geometry/EcalMapping
>> Entering Package Geometry/EcalTestBeam
>> Compile sequence completed for CMSSW CMSSW_13_1_X_2023-03-26-0000
gmake: *** [There are compilation/build errors. Please see the detail log above.] Error 1
+ eval scram build outputlog '&&' '(python3' /data/cmsbld/jenkins/workspace/ib-run-pr-tests/cms-bot/buildLogAnalyzer.py --logDir /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_13_1_X_2023-03-26-0000/tmp/el8_amd64_gcc11/cache/log/src '||' 'true)'
++ scram build outputlog
>> Entering Package Geometry/EcalMapping
Entering library rule at src/Geometry/EcalMapping/plugins
>> Compiling edm plugin /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_13_1_X_2023-03-26-0000/src/Geometry/EcalMapping/plugins/SealModule.cc
</pre>

--- a/Geometry/EcalTestBeam/test/testEcalTBCrystalMap.cpp
+++ b/Geometry/EcalTestBeam/test/testEcalTBCrystalMap.cpp
@@ -18,12 +18,12 @@ int main() {
 
     theTestMap.findCrystalAngles(thisCrystal, thisEta, thisPhi);
 
-    std::LogVerbatim("EcalTestBeam") << "Crystal number " << thisCrystal << " eta = " << thisEta
+    edm::LogVerbatim("EcalTestBeam") << "Crystal number " << thisCrystal << " eta = " << thisEta
                                      << " phi = " << thisPhi;
 
     int checkThisCrystal = theTestMap.CrystalIndex(thisEta, thisPhi);
 
-    std::LogVerbatim("EcalTestBeam") << "(eta,phi) = " << thisEta << " , " << thisPhi
+    edm::LogVerbatim("EcalTestBeam") << "(eta,phi) = " << thisEta << " , " << thisPhi
                                      << " corresponds to crystal n. = " << checkThisCrystal;
   }
 

